### PR TITLE
Fix Issue 20280 - Compiler segfault when building two small files

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: test/compilable/test20280.d [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -Icompilable/extra-files
+// EXTRA_FILES: extra-files/example7190/controllers/HomeController.d
```
Comment (Geod24) [inline test/compilable/test20280.d]: Doesn't look like this file is in this PR

**Comment by RazvanN7** [comment]
I think that the buildkite failure is unrelated to this patch.

**Review by ghost — APPROVED** [review]

**Comment by thewilsonator** [comment]
They seem annoyingly consistent, but they do look unrelated. Maybe try rebasing?

**Comment by wilzbach** [comment]
> I think that the buildkite failure is unrelated to this patch.

Agreed.

**Pull request merged**